<?xml version="1.0"?>
<launch>

    <arg name="node_frequency" default="20.0" />
    
    <!-- uncomment when Torsten global planner is merged into hydro... -->
    <!-- arg name="global_plan_topic" default="/move_base/GlobalPlannerWO/plan" /-->
    <arg name="global_plan_topic" default="/move_base/GlobalPlanner/plan" />

    <node pkg="mcr_navigation_tools" type="path_length_calculator" name="path_length_calculator" output="screen" ns="mcr_navigation" >
        <!-- control node frequency -->
        <param name="node_frequency" value="$(arg node_frequency)" />

        <!-- Remap to listen to desired global plan -->
        <remap from="~plan" to="$(arg global_plan_topic)"/>
    </node>

</launch>